packer.config.json

```diff
@@ -0,0 +1,27 @@
+{
+    "package_name": "gdk-for-unity-fps-starter-project",
+    "version": "0.1.3",
+    "git_packages": [
+        {
+            "clone_url": "git@github.com:spatialos/gdk-for-unity-fps-starter-project.git",
+            "clone_dir": "gdk-for-unity-fps-starter-project",
+            "exclude_paths": [
+                ".git",
+                ".github",
+                "ci",
+                "scripts"
+            ]
+        },
+        {
+            "clone_url": "git@github.com:spatialos/gdk-for-unity.git",
```

Comment:
For the next iteration, it definitely makes sense to allow for a reference to a specific packer file in the cloned repo, rather than repeating things like `exclude_paths` here

Reply:
Agreed - although given the longer term plan for delivery (i.e. - UPM) we won't have the sideloaded packages and will be less of a concern.